doc/INSTALL

```diff
@@ -246,353 +217,375 @@ General notes for the installation of the 4.2 release of KVIrc
     to "/usr/lib/libgsm.a", you might create the shared library manually
```

Comment:
I don't think it's very good idea to modify /usr/lib by hand for this.
1. This won't work if the library was compiled without `-fPIE`.
2. Files in /usr/lib should really be managed by package manager of the distro. Whenever package containing libgsm.a upgrades, the manually created libgsm.so stays old.
3. Distro can provide a better way to install libgsm.so. E.g. in Gentoo many packages have `static-libs` USE-flag which tells the package to install not only `.so`, but also `.a` (which is kinda the reverse, but the point still stands)